.gitignore

```diff
@@ -25,6 +25,7 @@ target/
 *~
 
 .gradle
+.vscode/
```

Comment:
you probably want to keep ths .vscode folder around

Reply:
push it? or just dont delete it?

Reply:
yeah push it, else it'll always appear as a change